configs/katello/nightly.yaml

```diff
@@ -9,6 +9,10 @@
     based_off: null
     arches:
       - x86_64
+  - name: katello-thirdparty-candlepin-el8
+    based_off: null
+    arches:
+      - x86_64
```

Comment:
I think we will need to add `    build_target: katello-nightly-el8-build` as that is how we have EL7 setup. This is not properly captured here in the config. This will mean also adding the katello-nightly-el8 tag setup with this PR.

Reply:
ack

Reply:
For EL7, this should actually look like:

```
 :tags:
   - name: katello-thirdparty-candlepin-rhel7
     based_off: null
+    build_target: katello-nightly-rhel7-build
     arches:
       - x86_64
   - name: katello-nightly-rhel7
```

The katello-thirdparty-candlepin-rhel7 tag builds using the katello-nightly-rhel7-build target. The EL8 environment should reflect the same structure.